lib/queryBuilder/operations/InsertOperation.js

```diff
@@ -27,8 +27,8 @@ class InsertOperation extends QueryBuilderOperation {
   }
 
   async onBefore2(builder, result) {
-    if (this.models.length > 1 && !isPostgres(builder.knex())) {
-      throw new Error('batch insert only works with Postgresql');
+    if (this.models.length > 1 && !isPostgres(builder.knex()) && !isMsSql(builder.knex())) {
```

Comment:
What do you think about creating a function to wrap this logic. Like `supportsBatchInsert`